### PR TITLE
upgrade to origami 2.0 specification

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: .github/workflows/test-origami-component.yml [hunk 1]
```diff
@@ -5,18 +5,11 @@ jobs:
...
-      id: repo_name
-      run: echo ::set-output name=REPO_NAME::${GITHUB_REPOSITORY/*\//}
-    - run: npx origami-ci branch
-      env:
-        ORIGAMI_CI_NAME: ${{ steps.repo_name.outputs.REPO_NAME }}
-        ORIGAMI_CI_VERSION: 0.0.0
-        GITHUB_TOKEN: ${{ secrets.ORIGAMI_FOX_ACCESS_TOKEN }}
-        ORIGAMI_PULL_REQUEST: ${{ github.event.pull_request.html_url }}
+    - run: npm install -g npm@^7
+    - run: npm install -g origami-build-tools@prerelease
```
Comment (chee) [inline .github/workflows/test-origami-component.yml]:
```suggestion
    - run: npm install -g origami-build-tools@11
```

File: package.json [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
+	"bugs": {
+		"url": "https://registry.origami.ft.com/components/o-syntax-highlight"
 	},
+	"license": "MIT",
+	"type": "module",
+	"browser": "main.js",
 	"scripts": {
 		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@^10\" install && npm_config_yes=true npx \"origami-build-tools@^10\" demo"
+		"build": "npm_config_yes=true npx \"origami-build-tools@prerelease\" install && npm_config_yes=true npx \"origami-build-tools@prerelease\" demo"
```
Comment (chee) [inline package.json]:
```suggestion
		"build": "npm_config_yes=true npx \"origami-build-tools@11\" install && npm_config_yes=true npx \"origami-build-tools@11\" demo"
```

**Comment by chee** [comment]
@Financial-Times/origami-core this'll need a README and a MIGRATION.md update, and then it'll be g-2-g

File: MIGRATION.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 ## Migration Guide
 
+### Migrating from v3 to v4
+
+V4 has dropped support for use through Bower. If you have been using Bower, you will need to follow the [bower to npm migration guide](https://origami.ft.com/docs/tutorials/bower-to-npm/).
+
+When using Sass, you will need to add `node_modules` to your `includePath` as well as either:
```
Comment (notlee) [inline MIGRATION.md:7]: Should this section also mention the namespace required on the JS import?

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v4.0.0 :tada:

File: MIGRATION.md (hunk 3 above)
Comment (chee) [reply MIGRATION.md:7]: actually i think I should remove this section, and just point people towards the guide!

**Pull request merged**